docs/design/kademlia-removal.md

```diff
@@ -0,0 +1,86 @@
+    separate discussion from this design document. This design is satisfied with a static trusted 
+    satellites list.
+    - Nodes will be able to communicate with satellites through the “transport” and “trust” packages 
+    directly rather than using kademlia to traverse the network to find the address of a given ID.
+
+### Network refreshes at a regular interval
+    - Nodes will keep themselves up to date in the network by pinging all the satellites in their 
+    trusted list every hour. 
+    - Satellites will ping the nodes back
+        - If is it successful, the satellite will insert or update the node in the overlay cache and 
+        notify the node of success
+        - If not, it does not proceed with updating the overlay cache, and the node receives an error message
+    - This will use grpc for now, but future development may use a non-ssl connection from node to satellite 
+    to initiate the refresh connection
+    - Update the overlay cache refresh method to communicate with nodes directly, without relying on 
+    kademlia lookups. Iterate through cache and dial each node.
+
+### Disintegrate Kademlia from the network, storj sim and testplanet setups
+    - Remove the “discovery” package
+    - Remove the kademlia transport observer
+    - Remove the bootstrap node - work with Ops
+    - Work with QA to make sure storage nodes don’t crash on errors related to the elimination of Kademlia 
+    if they don’t update immediately
+
+4.Update whitepaper to address kademlia removal and the addition of satellite opt-in
+    - Remove the audit gating design doc
+    - Update the wiki
+
+## Rationale
+Many peer-to-peer, decentralized systems employ the Kademlia implementation of a distributed hash table to 
+allow for locating peer nodes, exchanging messages and sharing data. However, due to the nature of our 
+network, we only use Kademlia for node discovery and address lookups given node IDs. This is useful when 
+satellites don’t know about all the nodes in the network and nodes are unfamiliar with all of the satellites 
+in the network. But with our recent business decision of simplifying the storage node operator user experience, 
+allowing nodes to directly select which satellites the would like to work with, we no longer require kademlia 
+for node discovery.
+
+## Implementation
+
+1. [Nodes should find satellites through the trust packages rather than kademlia](https://storjlabs.atlassian.net/browse/V3-2274)
+
+2. [Nodes keep up to date with satellites by periodically pinging and updating their trusted list](https://storjlabs.atlassian.net/browse/V3-2275)
+
+3. [Remove kad random lookups for discovery & update overlay cache refresh](https://storjlabs.atlassian.net/browse/V3-2305])
+
+4. [Disintegrate kademlia from the network, storj sim and testplanet setups](https://storjlabs.atlassian.net/browse/V3-2276)
+
+5. Update Documentation
```

Comment:
Should we also have an issue about removing vouchers?

Reply:
Ah yes definitely